apps/wallet-mobile/src/features/Swap/common/SwapInfoLink/SwapInfoLink.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import {Linking, StyleSheet, Text, TouchableOpacity} from 'react-native'
+
+import {useStrings} from '../strings'
+
+export const SwapInfoLink = () => {
+  const strings = useStrings()
+  return (
+    // TODO: add real link
```

Comment:
@SorinC6 if you can get the official link to zendesk.

Reply:
@SorinC6 Have asked in the slack about it https://emurgo.slack.com/archives/C05GB83MV1D/p1697540004073409?thread_ts=1697536526.322079&cid=C05GB83MV1 . as soon as will get the link will add it

Reply:
Isn't it
https://emurgohelpdesk.zendesk.com/hc/en-us/categories/4412619927695-Yoroi 

Not sure if there's one specific for swap, I can't find it on there.

Reply:
seems like it was not created yet - have added the yoroi webpage until we have an official zendesk link to add here